app/controllers/payment-links/metadata/metadata-form.js

```diff
@@ -80,4 +105,29 @@ function isNotEmpty (value) {
   return value && value.length !== 0
 }
 
+function isValidLengthForColumnHeader (value) {
+  return value && value.length !== 0 && value.length <= METADATA_MAX_HEADER_LENGTH
```

Comment:
I don't think the length validators should be checking for empty in this case as the error message for empty should be different. If for some reason this validator ran first it would give the message 'Column header must be 50 characters or fewer' even though the thing it failed on was having length 0. 

In practice it looks like `isNotEmpty` will always run first so this shouldn't need to be blocking, we could update it as part of a future PR.

Reply:
Good point - I will create another PR for this.
Thanks